apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/TableStatsCollectorUtil.java

```diff
@@ -190,22 +264,23 @@ private static long getManifestFilesCount(
   }
 
   /** Get all data files count depending on metadata type to query. */
-  private static Dataset<Row> getAllDataFilesCount(
+  private static Map<Integer, DataFilesSummary> getFileMetadataTable(
       Table table, SparkSession spark, MetadataTableType metadataTableType) {
-    return SparkTableUtil.loadMetadataTable(spark, table, metadataTableType)
-        .selectExpr(new String[] {"file_path", "file_size_in_bytes"})
-        .dropDuplicates("file_path", "file_size_in_bytes");
```

Comment:
is duplication of `file_path` not necessary? Seems this is no longer there in the new implementation

Reply:
i don't see that it's necessary, not seeing online that this table can contain dupes. i also don't see context on why it was added in the original blame.

@maluchari maybe you have more context?

i will add it back though to be sure

Reply:
duplicate file paths is required since same file can be referenced multiple types across snapshots.